packages/shared/lib/utils/error.manager.ts

```diff
@@ -46,7 +47,7 @@ class ErrorManager {
         }
     }
 
-    public async report(e: any, config: ErrorOptionalConfig = { source: ErrorSourceEnum.PLATFORM }) {
+    public report(e: unknown, config: ErrorOptionalConfig = { source: ErrorSourceEnum.PLATFORM }, tracer?: Tracer): void {
```

Comment:
I had to pass the tracer around because the `report` mechanism is used at very different place. Let me know if you think of another way to log to the span, but I think it's handy and could be reused inside the runner or persist.

Reply:
I think it is fine. There is definitely a lot of room for improvements wrt how we do error handling in general and across services